tests/labhub_test.py

```diff
@@ -55,9 +55,11 @@ def test_invite_cmd(self):
                                    'select from one of the')
 
         self.mock_team.is_member.return_value = False
-
```

Comment:
Why did you remove this line? It is an irrelevant change.

Reply:
Apologies for the mistake. Correcting at once.